docker-compose.yml

```diff
@@ -57,6 +57,7 @@ services:
             - ENABLE_PREJOIN_PAGE
             - ENABLE_WELCOME_PAGE
             - ENABLE_CLOSE_PAGE
+            - DEFAULT_LANGUAGE
```

Comment:
Can you please put it alphabetically sorted ?

Reply:
OK, it's done.